src/main/java/com/ramya/RamyaController.java

```diff
@@ -3,7 +3,6 @@
 public class RamyaController {
 	public static void main(String[] args) {
 		System.out.println("ramya");

```

Comment:
change name as rajini in system.out.println();

Reply:
done

Reply:
done